custom_components/anna/__init__.py

```diff
@@ -0,0 +1 @@
+REQUIREMENTS = ['haanna==0.6.2']
```

Comment:
Is this still required? I'm asking because the manifest also has a requirements line. I would suggest adding a description in here instead so that the file is not empty.
```suggestion
"""
Plugwise Anna component for HomeAssistant
"""
```

Reply:
I'll retry this part, I added the `__init__.py` yesterday with that statement (even with the manifest in).

Reply:
Seems to work, commit following